typing/typecore.ml

```diff
@@ -7548,6 +7548,7 @@ and type_argument ?explanation ?recarg env (mode : expected_mode) sarg
               |> Value.proj (Comonadic Areality)
               |> regional_to_global
               |> Locality.disallow_right,
+              ret_sort,
```

Comment:
I think this is right, because it looks like `args` cannot have any `Omitted`s. But this is a bit subtle (I had to check that fact), and so probably best to have a comment saying so. This would be easy to get wrong in the future.